collectors/ebpf.plugin/ebpf.h

```diff
@@ -113,6 +114,7 @@ typedef struct ebpf_tracepoint {
 #define NETDATA_EBPF_CHART_TYPE_STACKED "stacked"
 #define NETDATA_EBPF_MEMORY_GROUP "mem"
 #define NETDATA_EBPF_SYSTEM_GROUP "system"
+#define NETDATA_EBPF_APPS_GROUP "apps"
```

Comment:
You started to use the function `ebpf_create_charts_on_apps` that uses `apps` by default, so this line is not more necessary, please, remove it.

Reply:
Good point!

Reply:
https://github.com/netdata/netdata/pull/11470/commits/21397ffa67451420bfe1d8a0f57f6865578141b6